lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,12 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    punctuation = '!@#$%^&*()-_+=[{]};:\'"<>/?,.\\|/1234567890~`¬'
```

Comment:
Why does the variable named `puncturation` contain digits?

Reply:
can i just name it punctuation_and_digits?

Reply:
what about the create_language_profile function? Is there nothing to correct?